tests/analyze/test_plot_correlation.py

```
@@ -0,0 +1,187 @@
+    assert corr_fig.get_axes()[0].get_title(), \
+        "Wrong subtitle, expect vertical"
+
+
+def test_plot_crosscorr_histogram_defaults(correlation_matrix, vertical_feature_name, horizontal_feature_name):
+    """Test plotting a cross-correlation histogram from a correlation matrix"""
+    corr_fig = plotCrossCorrHistogram(correlation_matrix,
+                                      vertical_feature_name=vertical_feature_name,
+                                      horizontal_feature_name=horizontal_feature_name)
+    assert isinstance(corr_fig, Figure), \
+        "Wrong return type, expect a matplotlib Figure"
+    assert corr_fig.get_suptitle() == f"Distribution of Pearson Cross Correlations", \
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)